apis/kueue/v1beta1/fairsharing_types.go

```diff
@@ -49,4 +53,31 @@ type FairSharingStatus struct {
 	// weight of zero and is borrowing, this will return
 	// 9223372036854775807, the maximum possible share value.
 	WeightedShare int64 `json:"weightedShare"`
+
+	// admissionFairSharingStatus represents information relevant to the Admission Fair Sharing
+	AdmissionFairSharingStatus *AdmissionFairSharingStatus `json:"admissionFairSharingStatus,omitempty"`
 }
+
+type AdmissionFairSharingStatus struct {
+	// ConsumedResources represents the aggregated usage of resources over time,
+	// with decaying function applied.
+	// The value is populated if usage consumption functionality is enabled in Kueue config.
+	ConsumedResources corev1.ResourceList `json:"consumedResources,omitempty"`
+
+	// LastUpdate is the time when share and consumed resources were updated.
+	LastUpdate metav1.Time `json:"lastUpdate,omitempty"`
+}
+
+type AdmissionScope struct {
+	AdmissionMode AdmissionMode `json:"admissionMode,omitempty"`
+}
+
+type AdmissionMode string
+
+const (
+	// AdmissionFairSharing based on usage, with QueuingStrategy as defined in CQ.
+	UsageBasedAdmissionFairSharing AdmissionMode = "UsageBasedAdmissionFairSharing"
+
+	// AdmissionFairSharing is disabled for this CQ
+	NoAdmissionFairSharing AdmissionMode = "NoFairSharing"
```

Comment:
```suggestion
	NoAdmissionFairSharing AdmissionMode = "NoAdmissionFairSharing"
```
Suggestion  for consistency of of the const name and the string

Reply:
Done